### ISSUE TITLE
Support Ruby 2.4

### ISSUE THREAD

**Issue by mattr-** [github/janky#237]
We should be close to being able to support Ruby 2.4. Here's a list of the following things that I know about needing to be done before we can say we officially support Ruby 2.4:

- Upgrade activerecord to version 4.2 at the earlier. Using a 5.x version would be better.
- This section of code https://github.com/github/janky/blob/013b416af1307312b313a94ec77c1de2deaf05e8/lib/janky.rb#L56-L60 can be removed
- Upgrade any other gems that are known to have incompatibilities with Ruby 2.4

I'd also like to deprecate anything less than Ruby 2.4, so a warning similar to https://github.com/github/janky/blob/013b416af1307312b313a94ec77c1de2deaf05e8/lib/janky.rb#L1-L4
for Ruby 2.3 and below would be great!

cc: @perlun

**Comment by mattr-** [comment]
The test suite is now running with Ruby 2.4. Need to ensure that the app functions as well.

**Comment by perlun** [comment]
Sorry, but where did I come into the picture? 😄

**Comment by mattr-** [comment]
From a comment in another issue :smile: ➡️   https://github.com/github/janky/pull/236#issuecomment-341409318

**Comment by perlun** [comment]
Ah, thanks! :+1: :smile:

**Comment by mattr-** [comment]
We're now running on Ruby 2.4 as of current master. :tada:
